CHANGES.txt

```diff
@@ -64,6 +64,10 @@ Changes
 Fixes
 =====
 
+- Fixed an issue whereby attempting operations on closed tables/partitions
+  would result in an exception about missing privileges, rather than the
+  expected Unsupported Feature exception.
```

Comment:
it should be stated that this issue is only happening at the enterprise version were privileges are enabled